pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -136,6 +136,8 @@ typedef struct __PyObjectEncoder {
     int outputFormat;
     int originalOutputFormat;
 
+    const char *Object_getBigNumStringValue;
```

Comment:
Wherever you declare this object to be in this struct needs to match the subsequent definition. So if you place this here then down on L2199 need to make sure it appears in the same position when